<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-OOQ7SLD" failures="1" tests="14" name="Different testing Feature Test" time="12.455" errors="0" timestamp="10 Jul 2018 02:38:22 GMT">
  <testcase classname="com.test.ExceptionTimeOutTest" name="infinitloopTest" time="2.017">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.test.ExceptionTimeOutTest.infinitloopTest() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.test.ExceptionTimeOutTest.infinitloopTest() didn't finish within the time-out 2000
at java.base@9.0.1/java.io.FileOutputStream.writeBytes(Native Method)
at java.base@9.0.1/java.io.FileOutputStream.write(Unknown Source)
at java.base@9.0.1/java.io.BufferedOutputStream.flushBuffer(Unknown Source)
at java.base@9.0.1/java.io.BufferedOutputStream.flush(Unknown Source)
at java.base@9.0.1/java.io.PrintStream.write(Unknown Source)
at java.base@9.0.1/sun.nio.cs.StreamEncoder.writeBytes(Unknown Source)
at java.base@9.0.1/sun.nio.cs.StreamEncoder.implFlushBuffer(Unknown Source)
at java.base@9.0.1/sun.nio.cs.StreamEncoder.flushBuffer(Unknown Source)
at java.base@9.0.1/java.io.OutputStreamWriter.flushBuffer(Unknown Source)
at java.base@9.0.1/java.io.PrintStream.write(Unknown Source)
at java.base@9.0.1/java.io.PrintStream.print(Unknown Source)
at java.base@9.0.1/java.io.PrintStream.println(Unknown Source)
at app//com.test.ExceptionTimeOutTest.infinitloopTest(ExceptionTimeOutTest.java:11)
at java.base@9.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.base@9.0.1/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base@9.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base@9.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base@9.0.1/java.lang.Thread.run(Unknown Source)
... Removed 7 stack frames]]>
    </failure>
  </testcase> <!-- infinitloopTest -->
  <testcase classname="com.test.GoogleTitleTest" name="googleLogoTest" time="0.066"/>
  <testcase classname="com.test.GoogleTitleTest" name="googleTitleTest" time="0.035"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.012"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.003"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.005"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.0"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCountTest" name="sum" time="0.003"/>
  <testcase classname="com.test.TestNGFeatures" name="loginTest" time="0.001"/>
</testsuite> <!-- Different testing Feature Test -->
